Adamant/Modules/Settings/Notifications/NotificationSounds/NotificationSoundsView.swift

```diff
@@ -0,0 +1,130 @@
+//
+//  NotificationSoundsView.swift
+//  Adamant
+//
+//  Created by Yana Silosieva on 20.08.2024.
+//  Copyright © 2024 Adamant. All rights reserved.
+//
+
+import SwiftUI
+import CommonKit
+
+struct NotificationSoundsView: View {
+    @StateObject var viewModel: NotificationSoundsViewModel
+    
+    @Environment(\.dismiss) var dismiss
```

Comment:
Where do you set this param? Or is it a specific SwiftUI feature?

Reply:
yes, the @Environment(\.dismiss) parameter is a built-in  SwiftUI feature that allows to dismiss a view. It makes the code simpler..